### ISSUE TITLE
Social Sharing of Course Videos

### ISSUE THREAD

**Issue by ProductRyan** [openedx/platform-roadmap#245]
### Abstract

We believe that we’re leaving some of our best promotional content on the floor by keeping it behind an authentication wall. We also want to respect the course team’s ability to choose what videos should be available to public viewers.

### Context & Background

We believe that we’re leaving some of our best promotional content on the floor by keeping it behind an authentication wall. We also want to respect the course team’s ability to choose what videos should be available to public viewers.

### Scope & Approach

We intend to build a user experience that:
Allows course teams to select which video(s) they want to expose to the public
Allows learners to share video content to their social networks
Allows a new prospect, sourced from social media, to learn more about a course via engaging video content

This will require changes to Studio, the Learning MFE, and the Platform itself.

### Value & Impact

Initially we will observe the general adoption of the product, as well as the generated traffic from these socially-shared videos to Discovery pages.

We also hope to identify enrollment patterns/behaviors from prospects who first learn of edX and our course offerings from social media.

### Milestones and/or Epics

Milestone 1: Proof of Concept

Demo the ability to share a video to twitter and have the video preview in twitter and provide links back to the appropriate discovery page.

Milestone 2: MVP

Deploy the functionality to share to Twitter to a group of selected courses on edx.org who opt-in to a beta program. This includes the ability for beta testers to self-select the videos that they’d like to be public via Studio.

Milestone 3: General Availability 

Add more social sites that videos can be shared to and improve the data observability of the feature.

### Named Release

TBA

### Timeline

Q1-FY23

### Proposed By

2U

**Comment by pdpinch** [comment]
What's the status of this feature? I think I saw somewhere that it had shipped. 

Is there any end-user documentation at this time? How about operator documentation about enabling this feature?

**Comment by jmakowski1123** [comment]
I don't think this was ever developed beyond an edx.org specific feature. If someone is interested in picking this up and generalizing it for wider use, please open a new proposal.
